pkgs/misc/sndio/default.nix

```
@@ -12,6 +12,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = lib.optional stdenv.hostPlatform.isDarwin fixDarwinDylibNames;
   buildInputs = lib.optional stdenv.hostPlatform.isLinux alsa-lib;
 
+  postInstall = lib.optionalString stdenv.hostPlatform.isLinux "install -Dm644 contrib/sndiod.service $out/lib/systemd/system/sndiod.service";
```

```suggestion
  postInstall = ''
    lib.optionalString stdenv.hostPlatform.isLinux ''"install -Dm644 contrib/sndiod.service $out/lib/systemd/system/sndiod.service
  '';
```
There is no harm having that file on darwin, too.